modules/data_release/php/data_release.class.inc

```diff
@@ -247,6 +198,63 @@ class Data_Release extends \NDB_Menu_Filter
         return $userFiles;
     }
 
+    /**
+     * Function getFieldOptions
+     *
+     * @return array
+     */
+    protected function getFieldOptions() : array
+    {
+        $db = $this->loris->getDatabaseConnection();
+        return [
+            'users'     => $this->getUsersList($db),
+            'versions'  => $this->getVersionsList($db),
+            'filenames' => $this->getFilesList($db),
+        ];
+    }
+
+    /**
+     * Tells the base class that this page's provisioner can support
+     * the UserSiteMatch filter.
+     *
+     * @return bool always false
+     */
+    public function useSiteFilter() : bool
+    {
+        return false;
+    }
+
+    /**
+     * Tells the base class that this page's provisioner can support the
+     * HasAnyPermissionOrUserSiteMatch filter.
+     *
+     * @return ?array of site permissions or null
+     */
+    public function allSitePermissionNames() : ?array
+    {
+        return null;
+    }
```

Comment:
I may be mistaken, but this documentation comment does not seem to coincide with the documentation of the base method, no ? It seems to me that returning `null` means the provisionner does *not* support the `HasAnyPermissionOrUserSiteMatch` filter.

Other than that LGTM.

Reply:
Hi @MaximeMulder, thank you for the review. Very appreciated.

This is what I found in the declaration of the method.
![image](https://github.com/aces/Loris/assets/37309344/75f76ecd-247a-451a-96e4-e1d1759471e4)

The data release docs are not site tied. That's way I'm returning null in the implementation. The page don't use site filtering.

Reply:
Do you think it will be more accurate to specify this in the documentation of the implementation itself: That this page don't use site filtering ?

Reply:
It might be more accurate, or you could use the `{@inheritDoc}` annotation

Reply:
> Do you think it will be more accurate to specify this in the documentation of the implementation itself: That this page don't use site filtering ?

Sorry for the wait, but yes, just like Dave said, specifying either that or inheriting the parent documentation should do the job.

Reply:
sorry, somehow this escaped to my attention. Should be fixed now. Thank you.